db/migrate/20240522010424_add_pet_status_to_application_pets.rb

```diff
@@ -0,0 +1,5 @@
+class AddPetStatusToApplicationPets < ActiveRecord::Migration[7.1]
+  def change
+    add_column :application_pets, :pet_status, :string
+  end
+end
```

Comment:
This is exactly the way you want to do this! Great job figuring this one out!

To answer your question, I am not sure why you had to reset the database when adding this migration. You should have just been able to run the migration normally. The thing that might happen after running this migration is that some of your code would complain that it was missing the new column.